src/jlib.erl

```diff
@@ -77,13 +77,15 @@
 -type rsm_in() :: #rsm_in{}.
 -type rsm_out() :: #rsm_out{}.
 
+-type rfc3339_string() :: [byte(), ...]. %% calendar:rfc3339_string()
```

Comment:
please add a comment that `calendar` module doesn't export `rfc3339_string()` type